.gitignore

```diff
@@ -11,3 +11,6 @@ venv/
 
 # MacOS
 .DS_Store
+
+# output
+/data/output
```

Comment:
this should be just `data`, or `data/*`, or `data/input` and `data/output`

this file allows Git to ignore files from being tracked

with what you currently have, nothing will be ignored, because `/data` is not a folder you track locally